pkg/middleware/logging/logging.go

```diff
@@ -79,7 +80,18 @@ func (r *reporter) PostCall(err error, rpcDuration time.Duration) {
 		return
 	}
 
-	r.logger.Info(grpcReqCompleteKey, r.fields...)
+	var grpcService string
+	for _, field := range r.fields {
```

Comment:
Instead of looping thru all the fields to get the grpcService, will it be better to add this as part of [reporter constructor](https://github.com/openfga/openfga/blob/a77441226cbff1033623edc3b165afe12961b1e2/pkg/middleware/logging/logging.go#L149) ?

Reply:
Yeah I debated the two choices, initially was thinking to not duplicate info in the reporter, but it's a private struct anyway, so I think your suggestion makes sense.
IMO this is a bit nicer and cleaner with the v2 logging interceptors but that upgrade/change is a bit of a heavier lift.